pkg/controller/pxc/version.go

```diff
@@ -73,8 +189,8 @@ func (r *ReconcilePerconaXtraDBCluster) scheduleEnsurePXCVersion(ctx context.Con
 		err := r.client.Get(context.TODO(), types.NamespacedName{Name: cr.Name, Namespace: cr.Namespace}, localCr)
 		if k8serrors.IsNotFound(err) {
 			log.Info("cluster is not found, deleting the job",
-				"job name", jobName, "cluster", cr.Name, "namespace", cr.Namespace)
-			r.deleteEnsureVersion(jn)
+				"job name", jn, "cluster", cr.Name, "namespace", cr.Namespace)
```

Comment:
I would set `name` instead of `job name`. Also align other logs, somewhere there is `job name` and somewhere only `name` for `jn`.

Reply:
updated